workspace/resource_notebook_test.go

```diff
@@ -36,6 +36,42 @@ func TestResourceNotebookRead(t *testing.T) {
 		"language":       "PYTHON",
 		"id":             path,
 		"workspace_path": "/Workspace" + path,
+		"format":         "SOURCE",
+	})
+}
+
+func TestResourceNotebookReadWithState(t *testing.T) {
+	path := "/test/path.py"
+	objectID := 12345
+	qa.ResourceFixture{
+		Fixtures: []qa.HTTPFixture{
+			{
+				Method:   http.MethodGet,
+				Resource: "/api/2.0/workspace/get-status?path=%2Ftest%2Fpath.py",
+				Response: ObjectStatus{
+					ObjectID:   int64(objectID),
+					ObjectType: Notebook,
+					Path:       path,
+					Language:   "PYTHON",
+				},
+			},
+		},
+		Resource: ResourceNotebook(),
+		Read:     true,
+		New:      true,
+		State: map[string]any{
+			"source": "acceptance/testdata/tf-test-jupyter.ipynb",
+			"format": "JUPYTER",
```

Comment:
I don't think these will normally be part of the resource state during read for a newly imported resource. What is this test meant to cover?